### PR TITLE
[BAU][Vishal]: Updating Spark. Making repo user friendly.

### PR DESCRIPTION
**Pull request by svishal9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,40 +7,82 @@ These jobs are using _PySpark_ to process larger volumes of data and are suppose
...
+type could be local, colima or docker-desktop
+
+action could be unit-test, integration-test or job.
+
+Full list of commands for Mac and Linux users is as follows: 
+
+| S.No.      | Command | Action     |
+| :---:        |    :----   |          :--- |
+| 1      | ./go.sh lint       | Static analysis, code style, etc.   |
+| 2      | ./go.sh linting       | Static analysis, code style, etc.   |
```
Comment (sambhav13) [inline README.md:56]: gives error when tried locally 
./go.sh linting
Linting
./go.sh: line 135: ./scripts/linting.sh: No such file or directory

File: README.md [hunk 2]
```diff
@@ -7,40 +7,82 @@ These jobs are using _PySpark_ to process larger volumes of data and are suppose
...
+| 3      | ./go.sh install-with-docker-desktop       | Install the application requirements along with docker desktop   |
+| 4      | ./go.sh install-with-colima       | Install the application requirements along with colima   |
+| 5      | ./go.sh start-colima       | Start Colima   |
+| 6      | ./go.sh run-local-unit-test       | Run unit tests on local machine   |
```
Comment (sambhav13) [inline README.md:60]: Gives this error on running local unit test
>./go.sh run-local-unit-test
Running unit tests on local machine
./scripts/mac_or_linux/unit-test.sh: line 5: poetry: command not found

Comment (sambhav13) [reply README.md:60]: whenever running local poetry is not found error comes, is poetry installed as part of any other command?
>/go.sh run-local-job
Running job on local machine
./scripts/mac_or_linux/run-job.sh: line 5: poetry: command not found

File: README.md [hunk 3]
```diff
@@ -7,40 +7,82 @@ These jobs are using _PySpark_ to process larger volumes of data and are suppose
...
+
+type could be local, colima or docker-desktop
+
+action could be unit-test, integration-test or job.
+
+Full list of commands for Mac and Linux users is as follows: 
+
+| S.No.      | Command | Action     |
+| :---:        |    :----   |          :--- |
+| 1      | ./go.sh lint       | Static analysis, code style, etc.   |
```
[the remark this answers is not in the document]
Comment (svishal9) [reply README.md:55]: Thanks @sambhav13 . Fixed

File: README.md (hunk 1 above)
Comment (svishal9) [reply README.md:56]: Thanks @sambhav13 . Fixed

**Comment by srikar** [comment]
This ran without any errors for me. I ran the colima version and the local versions.

**Review by srikar — APPROVED** [review]
This ran without any errors for me. I ran the colima version and the local versions. @sambhav13 @svishal9

**Pull request merged**